examples/app/weather/templates/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title>{{ title if title else "Bokeh Plot" }}</title>
+        {{ bokeh_css }}
+        {{ bokeh_js }}
+    </head>
+    <body>
+        <div>
+            HERE IS SOME CUSTOM STUFF
+        </div>
+        {{ plot_div|indent(8) }}
+        {{ plot_script|indent(8) }}
```

Comment:
to me this is too obscure, I'd love to see it simplified down to `{{bokeh_head}}`, `{{document}}`, and `{{title}}` perhaps (or other more obviously-meaningful things). I believe it's possible to do that without breaking the ability to also do the above more detailed breakdown. But it could be its own PR for sure.